UnitTestFrameworkPkg/UnitTestFrameworkPkgCommon.dsc.inc

```diff
@@ -0,0 +1,22 @@
+## @file

```

Comment:
Why is the common dsc inc file part of the stack cookie change?  Seems like a different type of change.

Reply:
This was done to allow for target based unit tests and host based unit tests to have different stack cookie linker flags